README.adoc

```diff
@@ -393,15 +393,15 @@ you deem necessary, which might include the `SpanBuilder` interface. You can use
 creation and customization, including setting timestamps.
 
 SystemResource.java
-[source, Java, linenums, role="code_column"]
+[source, Java, linenums, role='code_column hide_tags=copyright']
 ----
 include::finish/system/src/main/java/io/openliberty/guides/system/SystemResource.java[tags=**;!copyright]
```

Comment:
- `tags=**;!copyright` should be removed, that's why hotspots for that file aren't working